examples/application_commands/localization.py

```diff
@@ -0,0 +1,96 @@
+from nextcord import Interaction, Locale, SlashOption
+from nextcord.ext import commands
+
+TESTING_GUILD_ID = 123456789  # Replace with your testing guild id
+
+
+bot = commands.Bot()
+
+
+@bot.slash_command(
+    name="hello",
+    name_localizations={
+        Locale.de: "hallo",
+        Locale.fr: "bonjour",
+        Locale.es_ES: "hola",
+    },
+    description="Description of the command in English",
+    description_localizations={
+        Locale.de: "Beschreibung des Befehls auf Deutsch",
+        Locale.fr: "Description de la commande en français",
+        Locale.es_ES: "Descripción del comando en español",
+    },
+    guild_ids=[TESTING_GUILD_ID],
+)
+async def hello_command(interaction: Interaction):
+
+    if interaction.locale == "de":
```

Comment:
Slight nitpick but could this unnecessary blank line be removed?

```suggestion
async def hello_command(interaction: Interaction):
    if interaction.locale == "de":
```